### PR TITLE
update prism commands

### PR DESCRIPTION
**Pull request by awburgard**
We shouldn't merge until we have the requisite deploy for spectral (whether next or latest)

This adds the following:

`generate:manifest` command to the package.json of _all_ generated components
`componentManifest` and `.spectral` to `prism integrations:init`

Updates the integrations `index.ts` to import and export the `componentRegistry`
Updates the `.tsconfig` to include the `.spectral` file

Removes `ConfigPages` generic from `flow` template

In order to copy over . files, we need to include the -all (or -a) flag, which includes files that start with a dot. Additionally, I added the exclude flag to make sure that non-.spectral files do not get added accidentally should someone open the templates in their finder. [docs](https://www.npmjs.com/package/copyfiles)

https://app.shortcut.com/prismatic/story/21537/update-prism-commands-for-components-init-and-integrations-init-for-manifests-and-their-usage?vc_group_by=day&ct_workflow=all&cf_workflow=500000005

### CODE REVIEW

File: src/commands/integrations/init/integration.ts [hunk 1]
```diff
@@ -107,6 +109,7 @@ export default class GenerateIntegrationCommand extends Command {
       scripts: {
         build: "webpack",
         import: "npm run build && prism integrations:import",
+        "generate:manifest": "npm run build && npx @prismatic-io/spectral component-manifest",
```
Comment (jasoncomes) [inline src/commands/integrations/init/integration.ts:112]: When you get a chance, lets remove this guy. We shouldn't be able to generate component manifest from CNI integrations.

File: templates/integration/src/componentRegistry.ts.ejs [hunk 2]
```diff
@@ -0,0 +1,3 @@
+import type { ComponentManifest } from "@prismatic-io/spectral"
```
Comment (jasoncomes) [inline templates/integration/src/componentRegistry.ts.ejs:1]:
```suggestion
import type { ComponentManifest } from "@prismatic-io/spectral";
```

File: templates/integration/src/componentRegistry.ts.ejs [hunk 3]
```diff
@@ -0,0 +1,3 @@
...
+
+export const componentRegistry = {}
```
Comment (jasoncomes) [inline templates/integration/src/componentRegistry.ts.ejs:3]:
```suggestion
export const componentRegistry = {};
```

**Review by jasoncomes — APPROVED** [review]

File: src/commands/components/init/component.ts [hunk 4]
```diff
@@ -116,6 +116,7 @@ export default class GenerateComponentCommand extends Command {
       scripts: {
         build: "webpack",
         publish: "npm run build && prism components:publish",
+        "generate:manifest": "npm run build && npx @prismatic-io/spectral component-manifest",
```
Comment (jasoncomes) [inline src/commands/components/init/component.ts:119]:
```suggestion
        "generate:manifest": "npm run build && npx @prismatic-io/spectral component-manifest --include-signature",
        "generate:manifest:dev": "npm run build && npx @prismatic-io/spectral component-manifest",
```
@bkegley do we want to provide them with a command that always generates a signature (if component published anyways) and another for purely development so the end user doesn't have to worry about the flags.

File: templates/integration/tsconfig.json.ejs [hunk 5]
```diff
@@ -8,5 +8,5 @@
     "module": "esnext",
     "moduleResolution": "node"
   },
-  "include": ["src"]
+  "include": ["src", ".spectral/*"]
```
Comment (jasoncomes) [inline templates/integration/tsconfig.json.ejs:11]:
```suggestion
  "include": ["src", ".spectral"]
```
Believe we only need the dir name here, like src.

Comment (bkegley) [reply templates/integration/tsconfig.json.ejs:11]: We need `*` for this to work.

**Pull request merged**